gtk/DetailsDialog.cc

```diff
@@ -1111,17 +1111,17 @@ class PeerModelColumns : public Gtk::TreeModelColumnRecord
         add(upload_rate_string);
         add(client);
         add(progress);
-        add(upload_request_count_int);
+        add(upload_request_count_number);
```

Comment:
I renamed these from `int` to `number` to indicate that they were an integral type but not necessarily an `int`; e.g. it could be a `size_t` or a `uint32_t`

Reply:
At Apple, they call "integer" what can refers to either int or long. Well, I mean "NSInteger" and "integerValue" and the likes. "Number" encompasses floating types too.

Reply:
That makes sense. Under those rules `number` still works here: the key difference is that we have two representations of the cell -- one is a display string for rendering, the other is a number for comparisons when sorting.